src/components/Textarea/Textarea.stories.mdx

```diff
@@ -61,6 +61,7 @@ The Textarea component defines a multi-line text input control.
       id: "textarea2",
       rows: "3",
       defaultValue: "Ubuntu",
+      onControlEnter: () => {alert("You Control Entered!")}
```

Comment:
Nit: can we make this a more easily-understandable phrase? ("You pressed control+enter")